--- create.php	2012-10-01 16:54:14.000000000 +0200
+++ create-idpcloud.php	2017-12-13 10:21:56.876760829 +0100
@@ -64,11 +64,11 @@

	else if ($action_number == 2)
		$redirect_url = sprintf('cmd.php?%s&template=%s&container=%s',
-			$href,'default',$request['template']->getContainerEncode());
+			$href,'custom_idpAccount:0',$request['template']->getContainerEncode());

	else
		$redirect_url = sprintf('cmd.php?%s&template=%s&dn=%s',
-			$href,'default',$request['template']->getDNEncode());
+			$href,'custom_idpAccount:0',$request['template']->getDNEncode());

	if ($action_number == 1 || $action_number == 2)
		printf('<meta http-equiv="refresh" content="0; url=%s" />',$redirect_url);
