--- ds_ldap_pla.php	2012-10-01 16:54:14.000000000 +0200
+++ /opt/phpldapadmin/lib/ds_ldap_pla.php	2017-06-20 11:52:12.000000000 +0200
@@ -16,7 +16,7 @@
 	function __construct($index) {
 		parent::__construct($index);
 
-		$this->default->appearance['password_hash'] = array(
+		$this->default->appearance['password_hash_custom'] = array(
 			'desc'=>'Default HASH to use for passwords',
 			'default'=>'md5');
 
